### PR TITLE
Translation of doc/index.md page 

### PR DESCRIPTION
**Pull request by abhaypawar**
The Index page is translated successfully. Thus, I request to you verify this contribution. Thankyou.
@spielers

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -4,31 +4,31 @@ description: The one stop location for tutorials, guides, and information about
...
 ---
 
-Gatsby is a blazing fast modern site generator for React.
+ गॅटस्बी ही एक रिऍक्ट साठी आधुनिक साइट जनरेटर आहे
 
-## Get Started
+## सुरुवात करा
 
-There are two main ways to get started with Gatsby:
+सुरुवाती साठी खालील दोन पर्यायांचा अवलंब करावा.
```
Comment (visshaljagtap) [inline docs/docs/index.md]: Sugggestion: गॅटस्बी सह प्रारंभ करण्याचे दोन मुख्य मार्ग आहेत

File: docs/docs/index.md [hunk 2]
```diff
@@ -4,31 +4,31 @@ description: The one stop location for tutorials, guides, and information about
...
 
-1. [Tutorial](/tutorial/): Step-by-step instructions on how to install Gatsby and start a project: written for people without Gatsby or web development experience, though these learning resources have helped developers of all skill levels.
-2. [Quick start](/docs/quick-start): One page summary of how to install Gatsby and start a new project.
+1. [Tutorial](/tutorial/): क्रमाक्रमाने गॅटस्बी कसे स्थापित करावे आणि प्रकल्प कसा सुरू करावा यावरील सूचना: गॅटस्बी किंवा वेब विकास अनुभवाशिवाय लोकांसाठी लिहिलेले असले तरी या शिक्षण स्त्रोतांनी सर्व कौशल्य पातळीच्या विकसकांना मदत केली आहे.
+2. [Quick start](/docs/quick-start): गॅटस्बी कसे स्थापित करावे आणि नवीन प्रकल्प कसा सुरू करावा याचा एक पृष्ठ सारांश.
 
-## Go further
+## प्रकल्प सुरु
```
Comment (visshaljagtap) [inline docs/docs/index.md]: पुढे जा

File: docs/docs/index.md [hunk 3]
```diff
@@ -4,31 +4,31 @@ description: The one stop location for tutorials, guides, and information about
...
+   - [Using Gatsby Professionally](/docs/using-gatsby-professionally/): कामावर इतरांना गॅटस्बी कसे समजावायचे यासाठी टिपा आणि युक्त्या जाणून घ्या, जेणेकरुन आपल्याकडे व्यावसायिकदृष्ट्या गॅटस्बीबरोबर काम करण्याची अधिक संधी असेल.
 
-3. Check out the [Ecosystem](/ecosystem/) libraries for Gatsby starters and plugins, as well as external community resources.
+3.गॅटस्बी स्टार्टर्स आणि प्लगइन्स तसेच बाह्य समुदाय संसाधनांसाठी [Ecosystem](/ecosystem/) लायब्ररी पहा.
 
-## Start contributing
+## योगदान देणे सुरू करा
 
-Visit the [Contributing](/contributing/) section to find guides on the Gatsby community, code of conduct, and how to get started contributing to Gatsby.
+गॅटस्बी समुदायाचे मार्गदर्शक, आचारसंहिता आणि गॅटस्बीला कसे सहयोग देणे सुरू करावे यासाठी मार्गदर्शक शोधण्यासाठी [Contributing](/contributing/) विभागास भेट द्या.
```
Comment (visshaljagtap) [inline docs/docs/index.md]: remove extra space

**Review by visshaljagtap — CHANGES_REQUESTED** [review]
@abhaypawar added comments

**Review by abhaypawar — COMMENTED** [review]
Removed as well as edited the flaws. Please check now.

**Review by visshaljagtap — APPROVED** [review]

**Comment by visshaljagtap** [comment]
Appreciated! @abhaypawar

**Pull request merged**